Podverse/MediaRef.swift

```diff
@@ -46,28 +46,37 @@ class MediaRef {
         
     }
     
-    static func retrieveMediaRefsFromServer(episodeMediaUrl: String? = nil, podcastFeedUrl: String? = nil, onlySubscribed: Bool? = nil, completion: @escaping (_ mediaRefs:[MediaRef]?) -> Void) {
+    static func retrieveMediaRefsFromServer(episodeMediaUrl: String? = nil, podcastFeedUrls: [String?] = [], onlySubscribed: Bool? = nil, page: Int? = 1, completion: @escaping (_ mediaRefs:[MediaRef]?) -> Void) {
```

Comment:
Do the podcastFeedUrls need to be optional? Is there a case where the array will contain a nil feedurl? Also, does page have to be an optional since it has a default value?

Reply:
Good call. Updated in next commit.